script/Common/00_00.txt

```diff
@@ -645,7 +648,7 @@ My ears no longer work properly,
 [sha:e988767973a8dd9a412569b16084d2ef1dcd5ad7]{
 -- Page 46, Offset 230.
 -- 止まない音は、火のついた<蝉|せみ>みたいで、不安になる。
-The never-ending noise she makes almost sounds like a cicada that has been set on fire.
+The never-ending noise she makes almost sounds like a cicada that's been set on fire.
  It makes me uncomfortable.
 }
 [sha:a345aa121412237e3cd0efd954d9ceb74891773c]{
```

Comment:
These 3 lines flow really weirdly.

Offset 246
>Ah, so that's why.

Offset 247

>That makes sense.

Offset 247

>Of course there wouldn't be a mirror here.